File: src/lib/software/medal.py
from src.lib.software import BaseSoftware

class Medal(BaseSoftware):
    def __init__(self):
        super().__init__()

        self.download_name = 'MedalSetup.exe'

    def resolve_download_url(self):
        self.url_resolved.emit('https://install.medal.tv')

File: packages/app/src/services/software/definitions/medal.ts
import { ok } from 'neverthrow';
import { SoftwareCategory } from 'shared';
import type { ISoftwareDefinition } from 'shared';

export default class implements ISoftwareDefinition {
	public key = 'medal';
	public name = 'Medal';
	public category = [SoftwareCategory.Gaming, SoftwareCategory.Social];
	public downloadName = 'MedalSetup.exe';
	public icon = 'medal.png';
	public homepage = 'https://medal.tv';

	public async resolveDownloadUrl() {
		return ok('https://install.medal.tv');
	}
}
